app/shared/models/setting.rb

```diff
@@ -0,0 +1,34 @@
+module FastlaneCI
+  # Setting model representing a single entry of a fastlane.ci system setting
+  class Setting
+    attr_accessor :key
+    attr_accessor :value
+    attr_accessor :verify_block
```

Comment:
just #fastlane things ❤️

Reply:
Haha yeah, we're slowly building `ConfigItem` again :trollface: